packages/rlc-common/src/buildClient.ts

```diff
@@ -86,8 +86,7 @@ export function buildClient(model: RLCModel): File | undefined {
     credentialKeyHeaderName,
     customHttpAuthHeaderName
   } = model.options;
-  const credentialTypes =
-    credentialScopes && credentialScopes.length > 0 ? ["TokenCredential"] : [];
+  const credentialTypes = credentialScopes ? ["TokenCredential"] : [];
```

Comment:
So here I just remove the `credentialScopes.length > 0` condition so that for empty scopes we could have `TokenCredential` in factory method.